### PR TITLE
Server testnet fixes

### PR DESCRIPTION
**Pull request by shahafn**

### CODE REVIEW

File: src/relayserver/RelayServer.js [hunk 1]
```diff
@@ -11,6 +11,13 @@ const StakeManagerABI = require('../common/interfaces/IStakeManager')
 const getDataToSign = require('../common/EIP712/Eip712Helper')
 const RelayRequest = require('../common/EIP712/RelayRequest')
 const utils = require('../common/utils')
+const Chains = {
+  mainnet: { chainId: 1, name: 'mainnet' },
```
Comment (drortirosh) [inline src/relayserver/RelayServer.js:15]: but `web3.eth.net.getNetworkType()` does it

Comment (shahafn) [reply src/relayserver/RelayServer.js:15]: Ok ty

File: src/relayserver/RelayServer.js [hunk 2]
```diff
@@ -338,6 +345,20 @@ class RelayServer extends EventEmitter {
...
+        chain = Chains.ropsten.name
+        break
+      case Chains.kovan.chainId:
+        chain = Chains.kovan.name
+        break
+      case Chains.mainnet.chainId:
+        chain = Chains.mainnet.name
+        break
+    }
+    this.rawTxOptions = { chain, hardfork: 'petersburg' }
```
Comment (drortirosh) [inline src/relayserver/RelayServer.js:360]: why petersbug? we're in istanbul, no?

Comment (shahafn) [reply src/relayserver/RelayServer.js]: fixed

**Pull request merged**